frontend/src/utils/icons.tsx

```diff
@@ -0,0 +1,90 @@
+import { Icon } from '@equinor/eds-core-react'
+import {
+    account_circle,
+    accessible,
+    fullscreen,
+    fullscreen_exit,
+    notifications,
+    chevron_down,
+    clear,
+    pause_circle,
+    stop_circle,
+    play_circle,
+    do_not_disturb,
+    check_circle_outlined,
+    error_outlined,
+    time,
+    warning_outlined,
+    autorenew,
+    pause_circle_outlined,
+    remove_outlined,
+    battery,
+    battery_charging,
+    battery_alert,
+    battery_unknown,
+    cloud_off,
+    image,
+    arrow_back,
+    add,
+    history,
+} from '@equinor/eds-icons'
+
+Icon.add({
+    account_circle,
+    accessible,
+    fullscreen,
+    fullscreen_exit,
+    notifications,
+    chevron_down,
+    clear,
+    pause_circle,
+    stop_circle,
+    play_circle,
+    do_not_disturb,
+    check_circle_outlined,
+    error_outlined,
+    time,
+    warning_outlined,
+    autorenew,
+    pause_circle_outlined,
+    remove_outlined,
+    battery,
+    battery_charging,
+    battery_alert,
+    battery_unknown,
+    cloud_off,
+    image,
+    arrow_back,
+    add,
+    history,
+})
+
+export enum IconEnum {
```

Comment:
Can we name it just "Icons" or does this conflict with other libraries? 
```suggestion
export enum Icons {
```